src/robusta/utils/base64_utils.py

```diff
@@ -0,0 +1,14 @@
+import base64
+import logging
+
+
+def is_base64_encoded(value: str) -> bool:
+    try:
+        if base64.b64encode(base64.b64decode(value)) == bytes(value, 'utf-8'):  # key is b64
+            return True
+        else:
+            return False
+    except Exception as e:
+        logging.error("Error during base64 check")
```

Comment:
I don't think we need to print anything here
It's not a real error
When we get a value which isn't b64, we expect that to happen. It's just a way to test if value is b64 encoded
Printing this as errors will just spam the logs

Reply:
Fixed